api/secrets.go

```diff
@@ -184,3 +195,25 @@ func IsValidSecretType(secretType string) error {
 	}
 	return nil
 }
+
+// checkClustersBeforeDelete returns error if there's a running cluster that created with the given secret
+func checkClustersBeforeDelete(orgId, secretId string) error {
+
+	filter := map[string]interface{}{
+		"organization_id": orgId,
+		"secret_id":       secretId,
+	}
+
+	if modelCluster, err := model.QueryCluster(filter); err != nil {
+		return nil
```

Comment:
Handle error properly here.

Reply:
When an error occours here that means no cluster found the database with the given params and the secret can be deleted so the method sould be return nil error here. But i add an info log before return